packages/genesys-spark-components/src/components/stable/gux-copy-to-clipboard/example.html

```diff
@@ -11,3 +11,25 @@ <h1>gux-copy-to-clipboard</h1>
 <gux-copy-to-clipboard>
   <div slot="content">(123) 456-7890</div>
 </gux-copy-to-clipboard>
+
+<h4 style="margin-top: 12px; margin-bottom: 0">Example with Overflow</h4>
+<div
+  style="
+    width: 300px;
+    max-height: 200px;
+    overflow: auto;
+    border: blue 1px solid;
+  "
+>
+  <gux-copy-to-clipboard>
+    <div slot="content">
+      cing elit. Ut mollis serus posuere a. Proin pel cing elit. Ut mollis serus
+      posuere a. Proin pel cing elit. Ut mollis serus posuere a. Proin pel cing
+      elit. Ut mollis serus posuere a. Proin pel cing elit. Ut mollis serus
+      posuere a. Proin pel cing elit. Ut mollis serus posuere a. Proin pel cing
+      elit. Ut mollis serus posuere a. Proin pel cing elit. Ut mollis serus
+      posuere a. Proin pellentesque suscipit erat, id maximus lorem consectetur
+      nec. Sed vel viverra.
+    </div>
+  </gux-copy-to-clipboard>
+</div>
```

Comment:
I don't think we should add this example because at the very least it is bad practice if not actually against the Design system.

Reply:
Agreed